### PR TITLE
TC-01

### PR DESCRIPTION
**Pull request by SachinKumar1606**
TC-01

### CODE REVIEW

File: Config/TestNG.xml [hunk 1]
```diff
@@ -1,16 +1,23 @@
  <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
 <suite name="All Test Cases" verbose="1" >
-    <test name="Verify user is logged in">
-        <parameter name="testRunId1" value="TCTC-1"/>
+<!--    <test name="Verify user is logged in">-->
+<!--        <parameter name="testRunId1" value="TCTC-1"/>-->
+<!--        <classes>-->
```
Comment (SachinKumar1606) [inline Config/TestNG.xml]: Why Commented code is pushed?

File: Config/TestNG.xml [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
-            <class name="Test.LoginTest"/>
-        </classes>
-    </test>
-    <test name="Verify user is added">
-        <parameter name="testRunId" value="TCTC-2"/>
-        <classes>
-            <class name="Test.BaseClass"/>
+            <class name="Test.LeavePageTest"/>
         </classes>
+
```
Comment (SachinKumar1606) [inline Config/TestNG.xml]: Why extra space is here

File: src/test/java/Pages/AddUserDetails.java [hunk 3]
```diff
@@ -42,7 +42,7 @@ public void User() throws InterruptedException{
         driver.findElement(employeeName).sendKeys("Lisa");
         Thread.sleep(3000);
```
Comment (SachinKumar1606) [inline src/test/java/Pages/AddUserDetails.java]: Why hard wait?

File: src/test/java/Pages/LeavePage.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LeavePage {
```
Comment (SachinKumar1606) [inline src/test/java/Pages/LeavePage.java:11]: In this page coding standers are not followed Revised you whole class before pushing it again

File: src/test/java/Pages/LeavePage.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+    public LeavePage (WebDriver driver){
+       this.driver = driver;
+       wait =new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
+
+    public void LeaveVerification () {
+        driver.findElement(Leave).click();
+        driver.findElement(apply).click();
+//        wait.until(ExpectedConditions.elementToBeClickable(LeaveType));
```
Comment (SachinKumar1606) [inline src/test/java/Pages/LeavePage.java]: Why commented code?

File: src/test/java/Pages/LeavePage.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        driver.findElement(LeaveType).click();
+        driver.findElement(leaveTypeDropdown).click();
+        driver.findElement(FromDate).sendKeys("2022-09-08");
+        wait.until(ExpectedConditions.elementToBeClickable(Comment));
+        driver.findElement(Comment).sendKeys("Leave for 3 days");
+
```
Comment (SachinKumar1606) [inline src/test/java/Pages/LeavePage.java]: Blank line?

File: src/test/java/Pages/PageFactory.java [hunk 7]
```diff
@@ -14,6 +14,8 @@ public class PageFactory {
 
     private AdminUserRegistration AdminUserRegistration;
 
+    private LeavePage LeavePage;
```
Comment (SachinKumar1606) [inline src/test/java/Pages/PageFactory.java]: it should be like 
private LeavePage leavePage;

File: src/test/java/Test/LoginTest.java [hunk 8]
```diff
@@ -10,7 +10,8 @@ public void LoginTest() throws InterruptedException{
         pageFactory.getMenu().menu();
         pageFactory.getAdduser().start();
         pageFactory.getAddUserDetails().User();
```
Comment (SachinKumar1606) [inline src/test/java/Test/LoginTest.java:15]: Test name do not match the test which we are writing in this class

File: Config/TestNG.xml (hunk 1 above)
Comment (Sourav12100) [reply Config/TestNG.xml:5]: I was unaware about not pushing the commented code, Did comment this part of code in order to run only LeavePageTest

Comment (Sourav12100) [reply Config/TestNG.xml]: @SachinKumar1606 I was unaware about not pushing the commented code, Did comment this part of code in order to run only LeavePageTest

File: Config/TestNG.xml (hunk 2 above)
Comment (Sourav12100) [reply Config/TestNG.xml]: I have resolve the spaces

File: src/test/java/Pages/LeavePage.java (hunk 4 above)
Comment (Sourav12100) [reply src/test/java/Pages/LeavePage.java:11]: Revised Standers in a proper way

File: src/test/java/Pages/LeavePage.java (hunk 5 above)
Comment (Sourav12100) [reply src/test/java/Pages/LeavePage.java]: I was unaware about not pushing the commented code , now I have removed the commented code

File: src/test/java/Pages/LeavePage.java (hunk 6 above)
Comment (Sourav12100) [reply src/test/java/Pages/LeavePage.java]: I have removed the blank line

File: src/test/java/Pages/PageFactory.java (hunk 7 above)
Comment (Sourav12100) [reply src/test/java/Pages/PageFactory.java]: I have done the change in it as given

File: src/test/java/Test/LoginTest.java (hunk 8 above)
Comment (Sourav12100) [reply src/test/java/Test/LoginTest.java:15]: I'm confuse in it 
If it does not match  then it is not even showing error

**Pull request closed without merging**